test/Microsoft.Azure.SignalR.Management.Tests/IInternalServiceHubContextFacts.cs

```diff
@@ -135,10 +165,10 @@ void assertAction(ConcurrentDictionary<HubServiceEndpoint, List<MessageVerifiabl
                 }
             }
 
-            await RunCoreAsync(serviceTransportType, testAction, assertAction);
+            await MockConnectionTestAsync(serviceTransportType, testAction, assertAction);
         }
 
-        private async Task RunCoreAsync(ServiceTransportType serviceTransportType, Func<ServiceHubContext, Task> testAction, Action<ConcurrentDictionary<HubServiceEndpoint, List<MessageVerifiableConnection>>> assertAction)
+        private async Task MockConnectionTestAsync(ServiceTransportType serviceTransportType, Func<ServiceHubContext, Task> testAction, Action<ConcurrentDictionary<HubServiceEndpoint, List<MessageVerifiableConnection>>> assertAction)
```

Comment:
why rename?

Reply:
Because this method is not for all the tests. I think `RunCoreAsync` seems like applicable to all the tests.

Reply:
better to do renaming in a different PR...